.github/workflows/discussion_ping.yml

```diff
@@ -19,9 +19,8 @@ jobs:
       discussion_repo: ${{ github.event.repository.full_name }}
     steps:
       - name: Send notification for new discussion
-        run: |
-          curl \
-            "$slack_url" \
-            -X POST \
-            -H 'Content-Type: application/json' \
-            -d '{"text":"New discussion opened by '"$discussion_author"' in '"$discussion_repo"': #'"$discussion_number"' - '"$discussion_title"'","blocks":[{"type":"section","text":{"type":"mrkdwn","text":":github: New discussion opened by *'"$discussion_author"'* in `'"$discussion_repo"'`:\n<'"$discussion_url"'|#'"$discussion_number"' - '"$discussion_title"'>"}}]}'
+        id: slack
+        uses: slackapi/slack-github-action@1.23.0
+        with:
+          payload: |
+            {"text": "New discussion opened by '$discussionAuthor' in '$discussionRepo': #$discussionNumber - '$discussionTitle'", "blocks": [{"type": "section", "text": {"type": "mrkdwn", "text": ":github: New discussion opened by *'$discussionAuthor'* in '`$discussionRepo`':\n<$discussionUrl|#$discussionNumber - '$discussionTitle'>"}}]}
```

Comment:
- The variables were prefixed with `$` because it was a Bash script, here we need to use GitHub's YAML interpolation syntax `${{ ... }}`.
- The payload can be formatted to make it more readable.

```suggestion
            {
              "text": "New discussion opened by ${{ env.discussionAuthor }} in ${{ env.discussionRepo }}: #${{ env.discussionNumber }} - '${{ env.discussionTitle }}'",
              "blocks": [
                {
                  "type": "section",
                  "text": {
                    "type": "mrkdwn",
                    "text": ":github: New discussion opened by *${{ env.discussionAuthor }}* in `${{ env.discussionRepo }}`:\n<${{ env.discussionUrl }}|#${{ env.discussionNumber }} - '${{ env.discussionTitle }}'>"
                  }
                }
              ]
            }
```

- The `slackapi/slack-github-action` action expects these two env vars to be set.
  - SLACK_WEBHOOK_URL
  - SLACK_WEBHOOK_TYPE

```yaml
env:
  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
  SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
```

You can refer to the `ci_cd.yml` workflow for an example of this action (at the bottom) and draw some inspiration from there.

Reply:
[7c71dda](https://github.com/WordPress/openverse/pull/3577/commits/7c71dda10b946a09e8a477efa53cc96cde5b943f)
[ad3b37e](https://github.com/WordPress/openverse/pull/3577/commits/ad3b37eba1962b66bbd3979055c9ba5fdd15acfc)

added required changes with above commits.